src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -837,10 +878,51 @@ public void onResize(ResizeEvent event)
          return;
       }
       
-      for (ChunkOutputUi output: outputs_.values())
+      // lightly debounce local resizes since they're somewhat expensive
+      resizePlotsLocal_.schedule(50);
+      
+      // heavily debounce remote resizes since they're very expensive
+      // (this actually spins up a separate R process to re-render all the
+      // plots at the new resolution)
+      resizePlotsRemote_.schedule(500);
```

Comment:
Yes, that's correct; it's 500ms measured from the time the user finishes resizing (as long as the size is actively changing the timer won't fire).